src/graph/unit_graph.cc

```diff
@@ -1492,4 +1527,14 @@ void UnitGraph::Save(dmlc::Stream* fs) const {
   }
 }
 
+UnitGraphPtr UnitGraph::Reverse() const {
+  CSRPtr new_incsr = out_csr_, new_outcsr = in_csr_;
+  COOPtr new_coo = nullptr;
+  if (coo_->defined()) {
+    new_coo = COOPtr(new COO(coo_->meta_graph(), aten::COOTranspose(coo_->adj())));
+  }
+  CHECK(restrict_format_ == SparseFormat::kAny);
```

Comment:
I think it's also tolerable if `restrict_format_` is `kCoo` and `coo_` is defined.

Reply:
I also think this check is unnecessary. It doesn't matter what is the restrict format of the original graph.